backend/scripts/format.sh

```
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd $SCRIPT_DIR
```

Shellcheck (0.9.0-1)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)